sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -245,6 +245,10 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
 
     (1 to 100).map(i => (i, i.toString)).toDF("key", "value").createOrReplaceTempView("testdata")
 
+    Seq((1, 1), (1, 2), (2, 1), (2, 2), (3, 1), (3, 2))
```

Comment:
previously we have 3 pre-loaded tables: `testdata`, `arraydata`, `mapdata`, which are key-value table, array type table and map type table. For the new join tests, I think only `lowerCaseData`, `upperCaseData`, `srcpart` make sense, why can't we use `testdata` for `testData2`, `src` and `src2`? They are all key-value tables.